projects/Rockchip/devices/TinkerBoard/options

```diff
@@ -28,3 +28,9 @@
 
   # kernel serial console
     EXTRA_CMDLINE="console=uart8250,mmio32,0xff690000 console=tty0"
+
```

Comment:
May be good to stick a comment in there letting others know why these are here...
``` suggestion

# Disabled Cores
```

Reply:
It's not permanent and this has to be done for all targets